src/crypto-api.ts

```diff
@@ -233,6 +233,24 @@ export interface CryptoApi {
      */
     bootstrapSecretStorage(opts: CreateSecretStorageOpts): Promise<void>;
 
+    /**
+     * Creates a new key backup version.
+     *
+     * If there are existing backups they will be replaced.
+     *
+     * The decryption key will be saved in Secret Storage (the `SecretStorageCallbacks.getSecretStorageKey` Crypto
+     * callback will be called)
+     * and the backup engine will be started.
+     */
+    resetKeyBackup(): Promise<void>;
+
+    /**
+     * Deletes the given key backup.
+     *
+     * @param version - The backup version to delete.
+     */
+    deleteKeyBackupVersion(version: string): Promise<void>;
```

Comment:
There is literally a section in this file called Secure Key backup: https://github.com/matrix-org/matrix-js-sdk/blob/b39741acbd66d6cd2455363412573f13e3f46747/src/crypto-api.ts#L357

Reply:
moved